src/debug-agent.cc

```diff
@@ -149,6 +154,9 @@ void Agent::Stop() {
   CHECK_EQ(err, 0);
 
   uv_close(reinterpret_cast<uv_handle_t*>(&child_signal_), nullptr);
+  // Close all remaining handles:
+  // uv_loop_close will return UV_EBUSY for handles which are not closed.
+  uv_walk(&child_loop_, close_handle, nullptr);
```

Comment:
They are the stdout and stderr pipes.  Closing them like that is not very optimal, it leaves their PipeWrap instances in an invalid state.

Proper cleanup is one of the things I have to tackle for the multi-isolate work.  I'll try to get around to it later this week.